### PR TITLE
pfile-utils (new formula)

### PR DESCRIPTION
**Pull request by Marvin182**
pfile-utils is a a collection of utilities for manipulating PFiles, the ICSI feature file archive format. It was invented to store training data for speech recognition system and neural networks and is still used by several research teams in the field.

quicknet is a software suite for training simple neural networks and required by pfile-utils.

### CODE REVIEW

File: Library/Formula/quicknet.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  end
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```
Comment (bfontaine) [inline Library/Formula/quicknet.rb]: Please write a test for this.

File: Library/Formula/quicknet.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  url "ftp://ftp.icsi.berkeley.edu/pub/real/davidj/quicknet.tar.gz"
+  version "3.33"
+  sha256 "035003767e1d7580ae30bdf3c4ff839a9b380e239d4b2ecde85df55d94f9a145"
+
+  stable do
+    patch do
+      url "https://gist.githubusercontent.com/Marvin182/41492c9d1992052fa35d/raw/7e8f3e4102f7cbb2408cf7bf07b2f01f6f1edcb3/patch_quicknet_mandir.diff"
+      sha256 "ebe0f7c57b549fcdef5a2945f50e2dc1145a966113ec6138d8f7ab139ec0b459"
+    end
+  end
```
Comment (bfontaine) [inline Library/Formula/quicknet.rb]: You can move the `patch do` block at top-level and remove `stable do` if the formula doesn’t have any other build type (`devel` and/or `head`).

File: Library/Formula/pfile-utils.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "pkg-config" => :build
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```
Comment (bfontaine) [inline Library/Formula/pfile-utils.rb]: Please write a test here too.

File: Library/Formula/quicknet.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
+class Quicknet < Formula
+  desc "Software suite for multi-layer perceptrons, designed for ASR."
+  homepage "http://www1.icsi.berkeley.edu/Speech/qn.html"
+  url "ftp://ftp.icsi.berkeley.edu/pub/real/davidj/quicknet.tar.gz"
+  version "3.33"
+  sha256 "035003767e1d7580ae30bdf3c4ff839a9b380e239d4b2ecde85df55d94f9a145"
+
+  stable do
+    patch do
+      url "https://gist.githubusercontent.com/Marvin182/41492c9d1992052fa35d/raw/7e8f3e4102f7cbb2408cf7bf07b2f01f6f1edcb3/patch_quicknet_mandir.diff"
```
Comment (bfontaine) [inline Library/Formula/quicknet.rb]: Please add a comment explaining why this patch is necessary as well as a link to the upstream submission. Thanks!

**Comment by bfontaine** [comment]
Would these be a better fit for our [`homebrew/science`](https://github.com/Homebrew/homebrew-science) tap?

**Comment by Marvin182** [comment]
The issues should be fixed and yes it might be a better match for homebrew/science. Should I open a new pull request?

**Comment by bfontaine** [comment]
You should open it at https://github.com/Homebrew/homebrew-science. Thanks!

**Comment by Marvin182** [comment]
done: https://github.com/Homebrew/homebrew-science/pull/3290

**Comment by Marvin182** [comment]
Ok, didn't fit in there either. Sad...

**Pull request closed without merging**